src/main/HttpError.ts

```diff
@@ -1,8 +1,10 @@
 export class HTTPError extends Error {
   status: number;
+  errors: unknown[]; // this is to play nicely with openapi-validator ValidationError
 
-  constructor(message: string, status: number) {
+  constructor(message: string, status: number, errors?: unknown[]) {
```

Comment:
If you use an access modifier like `private` you don't need to declare the property.

Also, you can assign default values rather than doing it in the constructor

```suggestion
  constructor(message: string, private readonly status: number, private readonly errors?: unknown[] = []) {
```

Reply:
tests show I am actually accessing those properties anyway lols. will set them to readonly at least.

Reply:
Remove the ? it will never be undefined if it has a default property